odh/base/monitoring/overrides/prometheus-operator/base/prometheus.yaml

```diff
@@ -9,6 +9,12 @@ metadata:
     app: prometheus
     prometheus: k8s
 spec:
+  alerting:
+    alertmanagers:
+      - name: alertmanager-operated
+        namespace: opf-monitoring
+        port: web
+  serviceAccountName: prometheus-operator
```

Comment:
@hemajv @HumairAK is this the SA we want to use or shouldwe use `prometheus-k8s`?

Reply:
If we can get it working with `prometheus-k8s` I think this is the proper way to go about it. AFAIK the `prometheus-operator` sa is for the operator, where as `prometheus-k8s` is for the instance.

Reply:
Was there a particular reason you need this SA @hemajv  ?

Reply:
updated the SA to `prometheus-k8s`. We need this SA so that prometheus is able to view the alertmanager